src/main/resources/commonsdivision/article/index.scala.html

```diff
@@ -0,0 +1,97 @@
+@import com.gu.contentatom.renderer.utils.GuardianDateFormatter
+@import com.gu.contentatom.thrift.atom.commonsdivision.Mp
+
+@(atom: com.gu.contentatom.thrift.Atom
+, data: com.gu.contentatom.thrift.atom.commonsdivision.CommonsDivision
+)
+
+@partyClassName(name: String) = @{
+  val shortName = name match {
+    case "Conservative" => "tory"
+    case "Labour" => "labour"
+    case "Democratic Unionist Party" => "dup"
+    case "Liberal Democrat" => "libdem"
+    case "Green Party" => "green"
+    case "Plaid Cymru" => "plaid"
+    case "Scottish National Party" => "snp"
+    case _ => "default"
+  }
+  s"atom--commonsdivision__party-$shortName"
+}
+
+@partyDisplayName(name: String) = @{
+  name match {
+    case "Democratic Unionist Party" => "DUP"
+    case "Scottish National Party" => "SNP"
+    case "Liberal Democrat" => "Lib Dem"
+    case "Green Party" => "Green"
+    case other => other
+  }
+}
+
+@mpsByParty(mps: Seq[Mp]) = {
+  <div class="atom--commonsdivision__party-list">
+  @mps.groupBy(_.party).map { case (partyName, partyMps) =>
+    <div>
+      <details class="atom--commonsdivision__party">
+        <summary>
+          <button class="atom__button atom--commonsdivision__party-handle atom__button--large atom__button--rounded division-party__handle">
+            <span class="atom--commonsdivision__party-button @partyClassName(partyName) is-on">
+              @fragments.icons.html.plus()
+              <span>
+                @partyDisplayName(partyName) - @partyMps.length
+              </span>
+            </span>
+            <span class="atom--commonsdivision__party-button @partyClassName(partyName) is-on">
+              @fragments.icons.html.minus()
+              <span>
+                @partyDisplayName(partyName) - @partyMps.length
+              </span>
+            </span>
+          </button>
+        </summary>
+        <div class="atom--commonsdivision__mps-list">
+          <ul>@partyMps.map { mp => <li>@mp.name</li>}</ul>
+        </div>
+      </details>
+    </div>
+  }
+  </div>
+}
+
+@for(title <- atom.title; description <- data.description) {
+
+  <details class="atom atom--snippet">
```

Comment:
Can we update the general `snippet` fragment so that your use case works? By adding a new code block in the signature perhaps